File: gif-bot/response.go
package main

import (
	"encoding/json"
)

type Response struct {
	Text string `json:"text"`
	Bot  string `json:"bot"`
}

func (r *Response) ToJson() ([]byte, error) {

	text, err := json.Marshal(r)
	if err != nil {
		return nil, err
	}
	return text, nil
}

File: noter-bot/src/main/java/com/gekoreed/entity/Response.java
package com.gekoreed.entity;

public class Response {
    public String text;
    public String bot = "sbot";

    public Response(String s) {
        this.text = s;
    }

    public static Response podcastStarted() {
        return new Response("Начинаю записывать заметки с этого момента.");
    }
}
